### PR TITLE
Weekly patch release v1.7.3

### PR DESCRIPTION
**Pull request by awaelchli**
Commit list (pl:1.7.x and all:next)
```
gh pr list -s merged -S 'merged:2022-08-16T22:30:00.000Z..2022-08-24T22:00:00.000Z' --json mergedAt,milestone,url,mergeCommit,title --jq 'sort_by(.mergedAt) | reverse | .[] | select((.milestone.title == "pl:1.7.x") or (.milestone.title == null) or (.milestone.title == "all:next")) | [.url, .mergeCommit.oid, .title] | join(" ")' --limit 100
```
```
https://github.com/Lightning-AI/lightning/pull/14380 0bd5703b8135fb6541c50c490ad39d30544ce67b [CI] Trick Bagua into installing appropriate wheel in GPU tests
https://github.com/Lightning-AI/lightning/pull/14363 2157a3b84d50c7e5c5310063a52aeed2efed0a0f Add a required job checker as an action (1/2)
https://github.com/Lightning-AI/lightning/pull/14348 bb634310e765e5e988220c7c8f421fb09b9dd0f0 [CI] Bump CUDA in Docker images to 11.6.1
https://github.com/Lightning-AI/lightning/pull/14345 99ba95a38e5468c6fd31aa4274b301c161be4dc0 fix imports of collections.abc for py3.10
https://github.com/Lightning-AI/lightning/pull/14298 7a617ec90e1566c763be8ac7a200af1e4025412c Add back support for logging in the gradient clipping hooks
https://github.com/Lightning-AI/lightning/pull/14337 9cf9bc582002a9fc3e0d3b890dc1aff9734cc870 Bump tj-actions/changed-files from 24 to 28
https://github.com/Lightning-AI/lightning/pull/13964 db1835a82c17ee685a10e2d065056740421b1fde Fix an issue to avoid the impact of sanity check on `reload_dataloaders_every_n_epochs` for validation
https://github.com/Lightning-AI/lightning/pull/14306 0b1a29b5eba423cec440ccc2761244f0d9efda4c Remove references to local app admin view
https://github.com/Lightning-AI/lightning/pull/14287 2dbdb00b03a74ba32fa5e30c3ecc383b5ee767a8 Bump actions/setup-python from 2 to 4
https://github.com/Lightning-AI/lightning/pull/14281 bbb406bce9493aeb104adb6aa40b9623201c2a29 Update ipython[all] requirement from <=8.1.1 to <8.4.1 in /requirements
https://github.com/Lightning-AI/lightning/pull/14296 a8c6e69b43d12a639c1f4de7abd9fac521e28d21 Fix wrong num padding for RichProgressBar
https://github.com/Lightning-AI/lightning/pull/14234 7879628a3a0ff0b7ef6b48f91bc1d2d4375389e2 Fix access to logger attribute when multiple loggers are used
https://github.com/Lightning-AI/lightning/pull/14276 bd55b56f56e9ecf10839ea11ef30ac158bc547e1 Update scikit-learn requirement from <=1.1.1,>0.22.1 to >0.22.1,<1.1.3 in /requirements
https://github.com/Lightning-AI/lightning/pull/14088 caa0d594b1a4e26237add1780cc11d65833e5090 Update torch requirement from <=1.12.0,>=1.9.* to >=1.9.0.a,<1.13.0 in /requirements
https://github.com/Lightning-AI/lightning/pull/14215 047f0aa1e47161ac5c1a8828ecbd8923936d8989 Fix type check for non-standard schedulers in horovod
https://github.com/Lightning-AI/lightning/pull/14200 401eb2c535f3b34b39313fbe58407fb8cbc56a24 Update tensorboard requirement from <2.10.0,>=2.9.1 to >=2.9.1,<2.11.0 in /requirements
https://github.com/Lightning-AI/lightning/pull/14217 7d1731096609adb97c3a686d1cb5000bbaea4832 Terminate process when main process raises error in ServableModuleValidator
```

Author list (pl:1.7.x and all:next):
```
gh pr list -s merged -S 'merged:2022-08-17T22:30:00.000Z..2022-08-24T22:00:00.000Z' --json author,milestone --jq '.[] | select((.milestone.title == "pl:1.7.x") or (.milestone.title == null) or (.milestone.title == "all:next")) | ["@" + .author.login] | join("")' --limit 100 | sort --ignore-case | uniq | xargs echo
```

cc @borda

### CODE REVIEW

File: src/pytorch_lightning/CHANGELOG.md [hunk 1]
```diff
@@ -34,6 +44,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed not preserving set attributes on `DataLoader` and `BatchSampler` when instantiated inside `*_dataloader` hooks ([#14212](https://github.com/Lightning-AI/lightning/pull/14212))
 
 
+- Fixed an issue to avoid the impact of sanity check on `reload_dataloaders_every_n_epochs` for validation ([#13964](https://github.com/Lightning-AI/lightning/pull/13964))
```
Comment (awaelchli) [inline src/pytorch_lightning/CHANGELOG.md:47]: this landed in the wrong place :)

Comment (rohitgr7) [reply src/pytorch_lightning/CHANGELOG.md]: removed. thanks :)

**Comment by rohitgr7** [comment]
> @rohitgr7 Some of the dependabot and ci

@awaelchli since they are touching app ci, I discussed with @Borda, and they are not essential.

**Comment by awaelchli** [comment]
Then shouldn't they be relabelled? 
`all:next` means it needs to be included in both app- and pl releases. 
@carmocca was pressing on this very strongly last time.

**Comment by rohitgr7** [comment]
okay.. let's include them

**Review by Borda — APPROVED** [review]
lgtm

**Pull request merged**